lib/python/src/bailo/helper/release.py

```diff
@@ -275,16 +273,34 @@ def delete(self) -> Any:
         self.client.delete_release(self.model_id, str(self.version))
         return True
 
+    @property
+    def version(self):
+        return self._version_obj
+
+    @version.setter
+    def version(self, value):
```

Comment:
I think it's best to make this private. Just to avoid any problematic updating.

Reply:
Done